monitor/stream.go

```diff
@@ -200,13 +200,7 @@ func StreamHealthCheck(server string, nopts []nats.Option, check *Result, opts S
 }
 
 func streamCheckMirror(si *api.StreamInfo, check *Result, opts StreamHealthCheckOptions, log api.Logger) {
-	if (opts.SourcesLagCritical <= 0 && opts.SourcesSeenCritical <= 0) || si.Config.Name == "" || si.Config.Mirror == nil {
-		return
-	}
-
-	if si.Config.Name == "" {
-		log.Debugf("CRITICAL: no configuration present")
-		check.Critical("no configuration present")
+	if (opts.SourcesLagCritical <= 0 && opts.SourcesSeenCritical <= 0) || si.Config.Name == "" || len(si.Sources) > 0 {
```

Comment:
it might be worth explaining why we check Len of sources here, its pretty weird even if right hehe

Reply:
Good shout, I've added a comment.